docs/create-a-business-component.md

```diff
@@ -0,0 +1,345 @@
+---
+id: create-a-business-component
+title: Create a Business Component
+---
+
+In Front Commerce we have seperated our components in two categories: the **UI**
+components available in the `ui` folders, and the **Business** components
+available in the `modules` and `pages` folders.
+
+> If you feel the need to understand why we went for this organization, feel
+> free to refer to the [components history page](#TODO-lien-history) first.
+
+In this section, we will build a **Business Component**. If you have already
+gone through the [Create an UI component](#TODO-lien-create-ui-component), the
+core concept is the same.
+
+## What is a Business component
+
+A Business component will reside in the `src/theme/modules` directory. Those
```

Comment:
Nit: I am not sure it is « will reside » is correct in this context.

Proposal: « should live »